bot.js

```diff
@@ -5,8 +5,13 @@ const {MAPLE_STORY_CLASSES} = require("./constants.json")
 const client = new Discord.Client();
 
 /* importing functions from the commands dir */
-var add = require('./commands/add.js');
-var remove = require('./commands/remove.js');
+const add = require('./commands/add.js');
```

Comment:
To make this cleaner: `require('/commands/add.js').add;` 
Then you don't get that weird add.add function call